api-catalog-ui/frontend/src/components/DetailPage/DetailPage.jsx

```diff
@@ -41,6 +41,10 @@ const FeedbackButton = React.lazy(loadFeedbackButton);
 export default class DetailPage extends Component {
     componentDidUpdate() {
         const { selectedContentAnchor } = this.props;
+        document.title =
+            this.props.tiles?.length > 0
+                ? `${this.props.tiles[0].title} | ${process.env.REACT_APP_API_PORTAL_SERVICE_TITLE}`
```

Comment:
Shouldn't this take the selected tile? Why is it trying by default to take the first one of the list? Is the first one always the selected one?

Reply:
I think list has always one value, the selected one. So I am taking it.